content/en/docs/concepts/services-networking/service.md

```diff
@@ -124,23 +161,25 @@ metadata:
   name: my-service
 subsets:
   - addresses:
-      - ip: 1.2.3.4
+      - ip: 192.0.2.42
     ports:
       - port: 9376
 ```
 
 {{< note >}}
-The endpoint IPs may not be loopback (127.0.0.0/8), link-local
-(169.254.0.0/16), or link-local multicast (224.0.0.0/24). They cannot be the
-cluster IPs of other Kubernetes services either because the `kube-proxy`
-component doesn't support virtual IPs as destination yet.
+The endpoint IPs _may not_ be: loopback (127.0.0.0/8 for IPv4, ::1/128 for IPv6), or
```

Comment:
you think it makes sense to s/may/cannot ?

Reply:
I'll try “must not”

Reply:
How's that?

Reply:
sounds good to me